### PR TITLE
Fix/bugbash002

### PR DESCRIPTION
**Pull request by danbider**
readme and gitignore modifications to the data_prep tutorial.
I do recommend someone to take another pass at the finetuning dataset creation -- it was less clear to me.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -122,6 +123,7 @@ venv.bak/
 
 # python venv installed in the dir, llmfoundry-venv
 *-venv
+*-env
```
Comment (vchiley) [inline .gitignore:126]:
```suggestion
```
is this a venv name you personally use?
I feel like most ppl just use `venv` or something more specific.

File: scripts/data_prep/README.md [hunk 2]
```diff
@@ -2,9 +2,13 @@
 
 This folder contains scripts for converting text data from original sources (HF, JSON) to the Mosaic [StreamingDataset](https://github.com/mosaicml/streaming) format for consumption by our training scripts. StreamingDataset is designed to make training on large datasets from cloud storage as fast, cheap, and scalable as possible. In particular, it is custom built for multi-node, distributed training for large models while maximizing correctness guarantees, performance, and ease of use.
 
+They following scripts will run on CPUs (no GPUs needed.) Execute them from CLI where the `python` command is recognized and `llm-foundry` dependencies installed. All scripts should run from the ./llm-foundry/scripts/data_prep` directory.
```
Comment (vchiley) [inline scripts/data_prep/README.md:5]:
```suggestion
They following scripts will run on CPUs (no GPUs needed). Execute them in an environment with `python` and `llm-foundry` dependencies installed. All scripts should run from the `./llm-foundry/scripts/data_prep` directory.
```

File: scripts/data_prep/README.md [hunk 3]
```diff
@@ -2,9 +2,13 @@
...
 
 ## Converting a pretraining dataset
-Using the `convert_dataset_hf.py` script...
+### HuggingFace data
```
Comment (vchiley) [inline scripts/data_prep/README.md:8]:
```suggestion
`convert_dataset_hf.py` usage for different dataset sources is described below.

### HuggingFace data
```

File: scripts/data_prep/README.md [hunk 4]
```diff
@@ -2,9 +2,13 @@
...
+In this example, we use the `convert_dataset_hf.py` script to convert a HuggingFace `c4` dataset into a `StreamingDataset`, using the `EleutherAI/gpt-neox-20b` tokenizer. The resulting directory is saved at `./llm-foundry/scripts/data_prep/my-copy-c4`.
+
+Currently supporting `c4` and `The Pile`.
```
Comment (vchiley) [inline scripts/data_prep/README.md:11]:
```suggestion
Currently supports `c4` and `The Pile`.
```

**Comment by danbider** [comment]
all resolved, I think we can merge?

**Review by vchiley — APPROVED** [review]

**Pull request merged**